libdevcore/CommonJS.cpp

```diff
@@ -45,6 +45,13 @@ bytes padded(bytes _b, unsigned _l)
 	return asBytes(asString(_b).substr(_b.size() - std::max(_l, _l)));
 }
 
+bytes paddedRight(bytes _b, unsigned _l)
+{
+	while (_b.size() < _l)
```

Comment:
Why not just _b.resize(_l); return _b; ?
Is it supposed to append zero bytes or '0' - zero character? It is not clear from the documentation comment.